.prettierignore

```diff
@@ -1 +1,2 @@
-contracts/AssetHolder.sol
\ No newline at end of file
+contracts/AssetHolder.sol
+contracts/ConsensusApp.sol
```

Comment:
@geoknee I am finding that `prettier-plugin-solidity` is helpful for me. Do you have another formatter that works is working better for you?

Reply:
I found that plugin pretty helpful too, until it started complaining about valid solidity code. I don't currently have an alternative, unfortunately.

Reply:
Would you mind sharing an example of the code that was causing the plugin to complain?

Reply:
If you remove files from `.prettierignore` and then run `yarn prettier:check` you will see what I mean:
<img width="985" alt="Screenshot 2019-09-09 at 19 21 57" src="https://user-images.githubusercontent.com/1833419/64556327-1cb81980-d337-11e9-8e95-0e242d429611.png">

It always seem to be confused by `abi.decode`:
<img width="985" alt="Screenshot 2019-09-09 at 19 20 39" src="https://user-images.githubusercontent.com/1833419/64556295-090cb300-d337-11e9-8216-68d6ab7363f6.png">

(The cursor is at line 42:65.)